recipes/supervised/meta.yaml

```diff
@@ -0,0 +1,69 @@
+
+{% set name = "mljar-supervised" %}
+{% set version = "0.10.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mljar-supervised-{{ version }}.tar.gz
+  sha256: fe53445a7033d5655c45374aa3d1cda8028ac8cc8bbbbbdd7d07fc04fc6836a6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - xgboost ==1.3.3
+    - catboost ==0.24.4
+    - category_encoders ==2.2.2
+    - cloudpickle ==1.3.0
+    - dtreeviz ==1.3
+    - joblib ==1.0.1
+    - lightgbm ==3.0.0
```

Comment:
In the `conda` universe exact pinnings are a disaster which should be avoided at all costs. Only if your package ***really*** does only work with a single exact version of a dependency should it be pinned.

As it is here this package will ~never be able to be installed in another environment with other packages with overlapping dependencies

Reply:
It's fine in the `pip` world as `pip` will happily ignore your dependency specifications (I believe it prints a warning now). `conda` strictly enforces the dependency specifications for all packages (including transitive dependencies) in you env. This can often lead to packages being unable to be installed if the pinnings are unnecessarily strict.

Reply:
I see this is however just following upstream :/

The right fix is probably to put in PRs against upstream to loosen the pinnings.

Reply:
☝️ IDT this needs to prevent this from getting in as-is but the pinnings are something that will likely need to be revisited.

Reply:
@dhirschfeld thank you for your tips, I will try to loosen the pinnings and try again!

Thank you for your help, I was afraid that I will be lost among bots.